Plugins/NativeCamera/Source/NativeVideo.cpp

```diff
@@ -149,13 +149,18 @@ namespace Babylon::Plugins
         if (!m_IsPlaying)
         {
             m_IsPlaying = true;
-            NativeCameraImpl->Open(m_width, m_height, m_frontCamera).then(m_runtimeScheduler, arcana::cancellation::none(), [this, env, deferred](const arcana::expected<void, std::exception_ptr>& result) {
+            NativeCameraImpl->Open(m_width, m_height, m_frontCamera).then(m_runtimeScheduler, arcana::cancellation::none(), [this, env, deferred](const arcana::expected<const Camera::CameraDimensions*, std::exception_ptr>& result) {
                 if (result.has_error())
                 {
                     deferred.Reject(Napi::Error::New(env, result.error()).Value());
                 }
                 else
                 {
+                    if (result.has_value()) {
```

Comment:
Is this check actually needed? If it doesn't have an error, it must have a value, right?